### PR TITLE
add bundleReference to the claim specification

### PR DESCRIPTION
**Pull request by adamreese**
ref: [New features in Claims specification require changes here](https://github.com/cnabio/libcnab-rust/issues/30)
https://github.com/cnabio/cnab-spec/pull/299

### CODE REVIEW

File: src/claim.rs [hunk 1]
```diff
@@ -26,6 +26,10 @@ pub struct Claim {
     pub result: Response,
     /// A ulid to track the revision
     pub revision: String,
+    /// A canonical reference to the bundle used in the last action.
```
Comment (glyn) [inline src/claim.rs:29]:
```suggestion
    /// An optional canonical reference to the bundle used in the last action.
```

File: src/claim.rs [hunk 2]
```diff
@@ -26,6 +26,10 @@ pub struct Claim {
...
+    /// This bundle SHOULD use a SHA to identify the specific version of the referenced bundle.
```
Comment (glyn) [inline src/claim.rs]:
```suggestion
    /// When provided, the bundle reference SHOULD be digested to identify a specific version of the referenced bundle.
```

File: src/claim.rs (hunk 1 above)
Comment (radu-matei) [reply src/claim.rs:29]: If this field is optional, could we represent it as `Option<String>` instead?

Comment (glyn) [reply src/claim.rs:29]: It either already is or it has already been updated to be so.

Comment (radu-matei) [reply src/claim.rs:29]: Oh, you're right.

**Review by technosophos — APPROVED** [review]
Yeah, with Glyn's minor doc changes, this is totally LGTM. Thanks!

**Review by radu-matei — APPROVED** [review]

**Review by glyn — CHANGES_REQUESTED** [review]
I think my remaining suggestion should be applied for consistency with the spec.

**Review by glyn — APPROVED** [review]

**Comment by radu-matei** [comment]
I think this needs a rebase, then we can merge.
Thanks!

**Pull request merged**